src/core/lib/promise/status_flag.h

```diff
@@ -230,6 +230,12 @@ class ValueOrFailure {
   ValueOrFailure(Failure) {}
   // NOLINTNEXTLINE(google-explicit-constructor)
   ValueOrFailure(StatusFlag status) { CHECK(!status.ok()); }
+  // NOLINTNEXTLINE(google-explicit-constructor)
```

Comment:
What's the use case for this one?

I'd prefer not to have it if possible - reason being that it silently loses information (the code and message of the status) -- preferably we'd only ever add information to errors or be very explicit about removing it.

Reply:
I think I forgot to undo this. Removed now.